e2e/pages/registration-and-edit-page.ts

```diff
@@ -46,12 +53,18 @@ export class RegistrationAndEditPage {
 
   async fillPatientRegistrationForm(formValues: PatientRegistrationFormValues) {
     const tryFill = (locator: Locator, value?: string) => value && locator.fill(value);
+    const focusDatePicker = (locator: Locator) => {
+      const focusElement = locator.getByRole('presentation');
+      return focusElement.focus();
+    };
```

Comment:
Since we don't reuse this, let's just put it there instead of making a function for it.